<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataManager</class>
 <widget class="QDialog" name="DataManager">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1008</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1008</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据管理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/CMainWindow/Resources/data.png</normaloff>:/CMainWindow/Resources/data.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_content" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>29</y>
     <width>1011</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	border: 1px solid rgba(22, 59, 102, 1);
	border-radius: 4px;
color: rgba(22, 59, 102, 1);
font-size:16px;</string>
         </property>
         <property name="text">
          <string>清空数据</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_out">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	border: 1px solid rgba(22, 59, 102, 1);
	border-radius: 4px;
color: rgba(22, 59, 102, 1);
font-size:16px;</string>
         </property>
         <property name="text">
          <string>导出数据</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_Code">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;</string>
         </property>
         <property name="placeholderText">
          <string>请输入物料号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_chooseTime">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	border: 1px solid rgba(22, 59, 102, 1);
	border-radius: 4px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/config/no_selected.svg</normaloff>
           <normalon>:/images/config/selected.svg</normalon>:/images/config/no_selected.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_StartDate">
         <property name="styleSheet">
          <string notr="true">	border: 1px solid #eeeeee;
	border-radius: 4px;
color:rgba(0, 0, 0, 0.25);</string>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_EndDate">
         <property name="styleSheet">
          <string notr="true">	border: 1px solid #eeeeee;
	border-radius: 4px;
color:rgba(0, 0, 0, 0.25);</string>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Query">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	border: 1px solid rgba(22, 59, 102, 1);
	border-radius: 4px;
	color: #FFFFFF;
    font-size: 14px;
	background-color:rgba(22, 59, 102, 1);</string>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true">	border: 1px solid #eeeeee;
	border-radius: 4px;

</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="rightMargin">
        <number>24</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	border: 1px solid rgba(22, 59, 102, 1);
	border-radius: 4px;
	color: rgba(22, 59, 102, 1);
    font-size: 14px;</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1021</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1008</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">	background: rgba(233, 238, 242, 1);
	border: 1px solid #eeeeee;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>4</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14px;
color:rgba(0, 0, 0, 1);</string>
    </property>
    <property name="text">
     <string>数据查询</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../other/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Query</sender>
   <signal>clicked()</signal>
   <receiver>DataManager</receiver>
   <slot>QueryData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ConnectToDB()</slot>
  <slot>QueryData()</slot>
  <slot>SetDateEnabled(bool)</slot>
  <slot>ClickedOnTableWidget(QModelIndex)</slot>
  <slot>ClearData()</slot>
 </slots>
</ui>
